core/src/main/java/gov/nist/secauto/metaschema/core/datatype/adapter/MetaschemaDataTypeProvider.java

```diff
@@ -189,6 +189,14 @@ public final class MetaschemaDataTypeProvider // NOPMD - Used for service initia
    */
   @NonNull
   public static final TokenAdapter TOKEN = new TokenAdapter();
+  /**
+   * TODO: Update with URL for docs from from metaschema-framework/metaschema#90.
+   * 
+   * The Metaschema <a href=
+   * "https://pages.nist.gov/metaschema/specification/datatypes/#tbd">TBD</a> data
+   * type instance.
+   */
+  public static final QNameAdapter QNAME = new QNameAdapter();
```

Comment:
```suggestion
  /**
   * The Metaschema <a href=
   * "https://pages.nist.gov/metaschema/specification/datatypes/#tbd">TBD</a> data
   * type instance.
   */
  // TODO: Update with URL for docs from from metaschema-framework/metaschema#90.
  public static final QNameAdapter QNAME = new QNameAdapter();
```
Don't want this in the javadoc.